src/MatrixClientPeg.js

```diff
@@ -111,6 +126,14 @@ class MatrixClientPeg {
         if (localStorage) {
             opts.sessionStore = new Matrix.WebStorageSessionStore(localStorage);
         }
+        if (window.indexedDB && localStorage) {
+            opts.store = new Matrix.IndexedDBStore(
+                new Matrix.IndexedDBStoreBackend(window.indexedDB),
```

Comment:
I sort of want to suggest passing a database name in here, so you can have 2 separate js-sdk based apps on the same domain without them stepping on each other's feet (although that implies you'd also pass it through from vector to react-sdk).